### PR TITLE
Adding brio-wu and MagLoopAdvection tables

### PR DESCRIPTION
**Pull request by meudnaes**
Draft pull request

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -8,11 +8,14 @@ Here are the results from the last run associated with
   ![briowu-rho](img/brio-wu_test_rho.png){: style="width:400px"}
   ![briowu-bd2](img/brio-wu_test_bd2.png){: style="width:400px"}
   ![briowu-ud](img/brio-wu_test_ud.png){: style="width:400px"}
-
+  
+  **[Tables](tables/brio-wu.md)**
```
Comment (guadabsb15) [inline docs/index.md]: Instead of naming the link Tables, it would be nice if we called it Comparison to reference

File: docs/index.md [hunk 2]
```diff
@@ -8,11 +8,14 @@ Here are the results from the last run associated with
...
 
 ## Advection of a Magnetic Field Loop 
 
   ![loop-pb](img/magnetic_pressure_loop_multiplot_test.png)
+  
+  **[Tables](tables/MagLoopAdvection.md)**
```
Comment (guadabsb15) [inline docs/index.md]: same as above, instead of Tables, let's name it Comparison to reference

**Review by guadabsb15 — CHANGES_REQUESTED** [review]
This looks good, I think we should only change the name of the links from tables, to something less generic :) as I suggested in my comments

**Review by guadabsb15 — APPROVED** [review]

File: docs/tables/.ipynb_checkpoints/MagLoopAdvection-checkpoint.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Brio-wu errors
```
Comment (guadabsb15) [inline docs/tables/.ipynb_checkpoints/MagLoopAdvection-checkpoint.md]: I think we don't need this file. Could you please remove it?

Comment (meudnaes) [reply docs/tables/.ipynb_checkpoints/MagLoopAdvection-checkpoint.md]: done!

**Pull request merged**